config/config.go

```diff
@@ -90,7 +90,9 @@ type Config struct {
 		AuthToken  string `password:"true" info:"The primary Auth Token for Twilio. Must be primary (not secondary) for request valiation."`
 		FromNumber string `public:"true" info:"The Twilio number to use for outgoing notifications."`
 
-		DisableTwoWaySMS bool `info:"Disables SMS reply codes for alert messages."`
+		DisableTwoWaySMS      bool     `info:"Disables SMS reply codes for alert messages."`
+		SMSCarrierLookup      bool     `info:"Perform carrier lookup of SMS contact methods (required for SMSFromNumberOverride). Extra charges may apply."`
+		SMSFromNumberOverride []string `info:"List of 'carrier=number' pairs, SMS messages to numbers of the provided carrier string (exact match) will use the alternate From Number."`
```

Comment:
The best way would be to copy from the carrier lookup page in the toolbox but I don't know of an API Twilio provides with a master list of supported carriers we could use to validate.

The format has to match the carrier name from the lookup API call exactly (since that's how it's detected).

Reply:
Gotcha, maybe that could be something we can add in the future; some form of lookup using a dropdown rather than `carrier=number`

Reply:
Yeah, we could potentially have a cached list of the carrier names we've seen (it won't be all of them) as a convenience auto-complete. Could be an enhancement once this is merged, I think we should keep this PR minimal for now.

Likewise we could query the list of FromNumbers from the Twilio API too and make this a distinct component instead of requiring the `carrier=number` format from the user.